virt/backup_restore/virt-restoring-vms.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: ASSEMBLY
+include::modules/virt-document-attributes.adoc[]
+[id="virt-restoring-vms"]
+= Restoring virtual machines]
+:context: virt-restoring-vms
+:velero-domain: velero.io
+
+toc::[]
+
+You restore an OpenShift API for Data Protection (OADP) `Backup` custom resource (CR) by creating a xref:../../virt/backup_restore/virt-restoring-vms.adoc#oadp-creating-restore-cr_virt-restoring-vms[`Restore` CR].
+
+You can add xref:../../virt/backup_restore/virt-restoring-vms.adoc#oadp-creating-restore-hooks_virt-restoring-vms[hooks] to the `Restore` CR to run commands in init containers, before the application container starts, or in the application container itself.
```

Comment:
You are correct that the parenthetical commas are not necessary. However, I added them because "the application container" is repeated. Without the commas, my eye has trouble distinguishing the different contexts of that phrase. I think the commas add clarity in this case.